src/kernels/composable_kernel/include/tensor_operation/blockwise_gemm_xdlops.hpp

```diff
@@ -133,17 +108,12 @@ struct BlockwiseGemmBlockABlockBThreadCTransANormalBNormalC_xdlops
         return make_ConstantMatrixDescriptor_packed(Number<total_reg_size>{}, Number<1>{});
```

Comment:
Do you still want to use this version of ```GetThreadMatrixCDescriptor```?

Since this function was originally written for non-xdlops GEMM. I think it should be rewritten for xdlops GEMM.Are you going to rewrite this function in  #355? If so, I think it's OK to keep it untouched in this PR

Reply:
We can leave this change to the PR (#335)